fuzzer_container.py

```diff
@@ -28,6 +30,36 @@ def run_in_container_and_get_output(cmd: typing.List[str]) -> bytes:
     )
 
 
+def copy_plugin_into_container(file_path: str) -> str:
+    file_name = basename(file_path)
+    new_file_path = f'/fuzzer/plugin_{file_name}'
+    subprocess.call(
+        [
+            "docker",
+            "cp",
+            file_path,
+            f"wordpress1:{new_file_path}"
+        ]
+    )
+    return new_file_path
+
+
+def install_plugin_from_file(file_path: str) -> None:
+    new_file_path = copy_plugin_into_container(file_path)
+    run_in_container(
+        [
+            "/fuzzer/nodebug.sh",
+            "php.orig",
+            "/wp-cli.phar",
+            "--allow-root",
+            "--activate",
```

Comment:
When installing from svn and from Wordpress repository, we separately install and activate, and here we do this at once. I don't remember the reason for separately installing and activating anymore but I think it's worth keeping it consistent.

Reply:
Yes I saw this, we can keep it separate for consistency. In that case I would also recommend to do the same with dependency installation.